models/answer.py

```diff
@@ -2,6 +2,8 @@
 
 
 class AnswerInput(InputBase):
+    page_slug: str
+    chunk_slug: str
     answer: str
 

```

Comment:
I think it was nice we added error handling for `textbook_name` v. `page_slug`. Could do something similar here.

```suggestion

    @root_validator
    def answer_textbook_attributes_check(cls, values):
        textbook_name = values.get("textbook_name")
        chapter_index = values.get("chapter_index")
        section_index = values.get("section_index")
        subsection_index = values.get("subsection_index")
        if textbook_name is not None and (chapter_index is None or section_index is None or subsection_index is None):
            raise ValidationError("If providing textbook_name, provide values for chapter_index, section_index, and subsection_index.")

        return values
```